python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -3492,8 +3492,16 @@ def test_dataframe_sort_index(
 @pytest.mark.parametrize("inplace", [True, False])
 @pytest.mark.parametrize("na_position", ["first", "last"])
 def test_dataframe_mulitindex_sort_index(
-    axis, level, ascending, inplace, ignore_index, na_position
+    request, axis, level, ascending, inplace, ignore_index, na_position
 ):
+    request.applymarker(
+        pytest.mark.xfail(
+            condition=axis in (1, "columns")
+            and ignore_index
+            and not (level is None and not ascending),
+            reason="https://github.com/pandas-dev/pandas/issues/56478",
```

Comment:
Should hopefully be fixed in pandas 2.2: https://github.com/pandas-dev/pandas/pull/56487